src/api/app/controllers/webui/package_controller.rb

```diff
@@ -120,7 +120,15 @@ def update
           flash[:success] = 'Package was successfully updated.'
           redirect_to package_show_path(project: @project, package: @package)
         end
-        format.js { flash.now[:success] = 'Package was successfully updated.' }
+        format.js do
+          if @package.saved_change_to_report_bug_url?
+            flash[:success] = 'Package was successfully updated.'
+            render js: "window.location='#{package_show_path(project: @project, package: @package)}'"
+            return
+          end
```

Comment:
Why this custom code for the report bug url? Isn't the previous code not enough?

Reply:
Nop, it is not enough. The "Report Bug" link belongs to the actions list, not to the `in-place-editing` element, which contains all the other details of a project or a package.

Reply:
![Screenshot from 2024-08-02 10-30-24](https://github.com/user-attachments/assets/572504ab-afd9-4bc3-9851-9c4c9660bfeb)

Reply:
Then it makes little sense to use the ajax route to simulate a page reload. You should update the content of the page from `app/views/webui/package/update.js.haml`

Reply:
Sure. Done.